src/test/java/apicurito/tests/steps/CommonSteps.java

```diff
@@ -148,12 +150,49 @@ public void createParameterOnPage(String param, String page, String isPlus, Data
         }
         CommonUtils.openCollapsedSection(pageElement, section);
 
-        if (Boolean.valueOf(isPlus)){
+        if (Boolean.valueOf(isPlus)) {
             pageElement.$(section).$$("icon-button").filter(attribute("type", "add"))
                     .shouldHaveSize(1).first().shouldBe(visible).$("button").click();
-        }else {
+        } else {
             pageElement.$(section).$$("a").filter(text(aName)).shouldHaveSize(1).first().shouldBe(visible).click();
         }
         CommonUtils.fillEntityEditorForm(table);
     }
+
+    @When("deploy apicurito into operatorhub and subscribe")
+    public void deployApicuritoIntoOperatorhubAndSubscribe() {
+        String disableOH = "https://gist.githubusercontent.com/mmajerni/1ea94f02a6b66dd86ebe63d8e8a9f15e/raw/29801d1e281f85193408bfb34d971edbddd67d63/OHdisable.yaml";
+        String operatorsource = "https://gist.githubusercontent.com/mmajerni/17e4c892f0685e0b9eb2b7051af0494f/raw/5b40b8b45ab9e24afd0c89be6fa0e6ab7b2fa10a/operatorsource.yaml";
+        String operatorGroup = "https://gist.githubusercontent.com/mmajerni/b170fcb959c046633f91dda693f3396e/raw/1b5f373abadc2decba00c07530a055311b9ad01f/OHoperatorgroup.yaml";
+        String subscription = "https://gist.githubusercontent.com/mmajerni/25523c2e5cd720d99730ec0b4cf0cb96/raw/818e9f3683a5e5de5e0582598d3523ca03c9b01b/OHsubscription.yaml";
```

Comment:
Wouldn't it be better to have these yaml files stored locally in `src/test/resources`?